packages/backend/src/server/api/endpoints/drive/files/create.ts

```diff
@@ -23,30 +21,15 @@ export const meta = {
 	kind: 'write:drive',
 
 	params: {
-		folderId: {
-			validator: $.optional.nullable.type(ID),
-			default: null,
-		},
-
-		name: {
-			validator: $.optional.nullable.str,
-			default: null,
-		},
-
-		comment: {
-			validator: $.optional.nullable.str.max(DB_MAX_IMAGE_COMMENT_LENGTH),
-			default: null,
-		},
-
-		isSensitive: {
-			validator: $.optional.bool,
-			default: false,
-		},
-
-		force: {
-			validator: $.optional.bool,
-			default: false,
+		type: 'object',
+		properties: {
+			folderId: { type: 'string', format: 'misskey:id', nullable: true, default: null, },
+			name: { type: 'string', nullable: true, default: null, },
+			comment: { type: 'string', nullable: true, maxLength: 512, default: null, },
```

Comment:
I think its not a good idea to remove the constant and add a magic number here
```suggestion
			comment: { type: 'string', nullable: true, maxLength: DB_MAX_IMAGE_COMMENT_LENGTH, default: null, },
```
also in other places where `DB_MAX_IMAGE_COMMENT_LENGTH` is used.